sys-power/ncpufreqd/ncpufreqd-2.4.ebuild

```diff
@@ -7,8 +7,8 @@ EAPI="5"
 inherit cmake-utils
 
 DESCRIPTION="Daemon controlling CPU speed and temperature"
-HOMEPAGE="http://projects.simpledesigns.com.pl/project/ncpufreqd/"
-SRC_URI="http://projects.simpledesigns.com.pl/get/${P}.tar.bz2"
+HOMEPAGE="https://bitbucket.org/nelchael/ncpufreqd"
+SRC_URI="https://bitbucket.org/nelchael/${PN}/get/${P}.tar.bz2"
```

Comment:
Tell you what.

    SRC_URI="http://dev.gentoo.org/~mgorny/dist/${P}.tar.xz"

Please let me know if I did everything right. And remember to update DEPEND for .tar.xz.